blocks/feed/feed.js

```diff
@@ -76,14 +76,24 @@ const resultParsers = {
         }
       });
       if (cardImage) {
-        row.push(cardImage);
+        if (result.path) {
+          const pathImg = document.createElement('a');
+          pathImg.href = result.path;
+          pathImg.append(cardImage);
+          row.push(pathImg);
+        } else row.push(cardImage);
+      }
+      if (result.featured === 'true' && block.classList.contains('featured')) {
+        const divFeatured = document.createElement('div');
+        divFeatured.innerHTML = `<h5>${featuredInnerText}</h5>`;
```

Comment:
Maybe just directly ewad it from `placeholders.featured;` instead of reading elsewhere and passing it here.

Reply:
If we put placeholders.featured in the hightlight section - then means we need to add the await action as well - 

const placeholders = await fetchPlaceholders(locale); which will further need to make the whole function async. Due to which its not displaying anything. Already tested that. That is why putting the above in an already existing main async function & then calling the Highlight block, accordingly.  

Please advise.

Reply:
Fetching placeholder from anywhere is equivalent. The `await` itself does not cause any delay, its what you do inside the `await` that may cause delay,